packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -149,16 +211,28 @@ If necessary for brand consistency, you can customize the toggle shape and icon.
 ![Different toggle shapes and icons.](./img/toggle-customizations.svg)
 </div>
 
-### Using the navigation menu
+### Using the conversation history menu
 
-The ChatBot navigation menu primarily contains a users' conversation history with the ChatBot. Clicking the menu icon opens a side drawer in the ChatBot window.
+The ChatBot history menu contains a log of a users' previous chats. Clicking the menu icon opens a side drawer in the ChatBot window.
 
-By clicking into the navigation menu, users can search through previous conversations and perform additional actions, such as sharing a conversation with others.
+By clicking into the history menu, users can search through previous conversations and perform additional actions, such as sharing a conversation with others.
 
 <div class="ws-docs-content-img">
 ![Conversation history with an options menu opened on a previous conversation.](./img/conversation-history.svg)
 </div>
 
+When the conversation history is still loading, display skeleton items:
+
+<div class="ws-docs-content-img">
+![Chat history items loading.](./img/loading-state.svg)
+</div>
+
+If there's an error loading the conversation history, display an error screen with steps for resolving the error:
+
+<div class="ws-docs-content-img">
+![Error state in chat history.](./img/error-state.svg)
```

Comment:
Text differs from code examples; not sure if this matters.

Placeholder: "Search previous conversations"

> Could not load chat history
To try again, check your connection and reload this page. If the issue persists, contact the support team.

Reply:
Ah good catch, ty! I do want to match up - I think the default value in Figma was just different, but I'll use this text that's already in the code